node/src/PipeTransport.ts

```diff
@@ -287,9 +287,9 @@ export class PipeTransport
 		const rtpParameters = ortc.getPipeConsumerRtpParameters(
 			producer.consumableRtpParameters, this.#data.rtx);
 
-		const internal = { ...this.internal, consumerId: uuidv4() };
 		const reqData =
 		{
+			consumerId             : uuidv4(),
```

Comment:
Ok so this means that from now on the routing information is just the corresponding to the entity responsible of taking the action, right? I assume it's the same in all requests.

Reply:
Exactly